modules/yieldlabBidAdapter.js

```diff
@@ -253,6 +260,44 @@ function createSchainString (schain) {
   return `${ver},${complete}${nodesString}`
 }
 
+/**
+ * Get content object from bid request
+ * First get content from bidder params;
+ * If not provided in bidder params, get from first party data under 'ortb2.site.content' or 'ortb2.app.content'
+ * @param {Object} bid
+ * @returns {Object}
+ */
+function getContentObject(bid) {
+  if (bid.params.iabContent && utils.isPlainObject(bid.params.iabContent)) {
+    return bid.params.iabContent
+  }
+
+  const globalContent = config.getConfig('ortb2.site') ? config.getConfig('ortb2.site.content')
```

Comment:
in the pr comments you refer to the ortb2.content object as distinct from ortb2.site.content, correct? Do you mean to gather the former as well?

Reply:
i was mistaken. the content object is under site or app. sorry for the confusion.

Reply:
thanks!